pkg/reconciler/revision/resources/deploy.go

```diff
@@ -81,6 +83,26 @@ var (
 		MountPath: concurrencyStateTokenVolumeMountPath,
 	}
 
+	varAnnotationVolume = corev1.Volume{
```

Comment:
I think this should be `podinfoAnnotationVolume` or `podinfoVolume` based on the value on 44 of `annotationsPodInfoDirPath`

(Ditto with the `...VolumeMount`)